### PR TITLE
CAD-1407 stylish haskell

### PR DESCRIPTION
**Pull request by CodiePP**
description
-----------

- [x] stylish Haskell reformating of source code

checklist
---------

- [ ] compiles (`cabal v2-build` or `stack build`)
- [ ] tests run successfully (`cabal v2-test` or `stack test`)
- [ ] documentation added
- [ ] link to an issue
- [ ] add milestone (the current sprint)

### CODE REVIEW

File: iohk-monitoring/src/Cardano/BM/Snocket.hs [hunk 1]
```diff
@@ -168,37 +162,18 @@ instance Show (AddressFamily addr) where
...
+    , getRemoteAddr :: fd -> m addr
+    , addrFamily    :: addr -> AddressFamily addr
+    , open          :: AddressFamily addr -> m fd
+    , openToConnect :: addr -> m fd
+    , connect       :: fd -> addr -> m ()
+    , bind          :: fd -> addr -> m ()
+    , listen        :: fd -> m ()
+    , accept        :: fd -> Accept addr fd
+    , close         :: fd -> m ()
+    }
```
Comment (deepfire) [inline iohk-monitoring/src/Cardano/BM/Snocket.hs:176]: @CodiePP, do we want those comments removed?

File: tracer-transformers/example2/Main.hs [hunk 2]
```diff
@@ -50,8 +50,10 @@ main = do
 --- demo code
 
 -- a level of verbosity
-data Verbosity = Silent | Normal | Verbose
-                 deriving (Eq, Ord, Show)
+data Verbosity = Silent
+    | Normal
+    | Verbose
+    deriving (Eq, Ord, Show)
```
Comment (deepfire) [inline tracer-transformers/example2/Main.hs:56]: This looks rather odd..

File: tracer-transformers/example2/Main.hs [hunk 3]
```diff
@@ -70,8 +72,10 @@ condVerbosity v = condTracingM (const <$> verbosityP v)
 -- The distinct message types that you want to expose. This is the set
 -- of observables.
 
-data MyMessages a = MyStart a | MyNormal a | MyWarning (Int, a)
-                  deriving Show
+data MyMessages a = MyStart a
+    | MyNormal a
+    | MyWarning (Int, a)
+    deriving Show
```
Comment (deepfire) [inline tracer-transformers/example2/Main.hs:78]: Similrly, the ADT variants are formatted oddly.. as well as in the remaining cases.

File: iohk-monitoring/src/Cardano/BM/Snocket.hs [hunk 4]
```diff
@@ -29,19 +29,13 @@ module Cardano.BM.Snocket
...
-                                  Socket
-                                , SockAddr (..)
-                                )
+import           Network.Socket (Family (AF_UNIX), SockAddr (..), Socket)
 import qualified Network.Socket as Socket
 #if defined(mingw32_HOST_OS)
 import           Data.Bits
-import qualified System.Win32            as Win32
+import qualified System.Win32 as Win32
+import qualified System.Win32.Async as Win32.Async
```
Comment (deepfire) [inline iohk-monitoring/src/Cardano/BM/Snocket.hs:37]: This feels like a regression -- previously aliases were aligned vertically, which provided a table-like feeling.

File: tracer-transformers/example2/Main.hs (hunk 2 above)
Comment (CodiePP) [reply tracer-transformers/example2/Main.hs:56]: :man_shrugging:  yes, right. can we turn off "stylish-haskell" for a region?

File: iohk-monitoring/src/Cardano/BM/Snocket.hs (hunk 1 above)
Comment (CodiePP) [reply iohk-monitoring/src/Cardano/BM/Snocket.hs:176]: I think yes, this is OK.

Comment (CodiePP) [reply iohk-monitoring/src/Cardano/BM/Snocket.hs:179]: found a way how to keep those comments albeit shorter on a single line

File: tracer-transformers/example2/Main.hs (hunk 2 above)
Comment (CodiePP) [reply tracer-transformers/example2/Main.hs:55]: it can't get better than that.

**Pull request merged**